### PR TITLE
Pushing bash script

### PR DESCRIPTION
**Pull request by jagdeepsason2021**

### CODE REVIEW

**Review by shaikh-rashid — APPROVED** [review]

**Review by amanjeev — APPROVED** [review]

File: htsget.sh [hunk 1]
```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
```
Comment (shaikh-rashid) [inline htsget.sh]: add `set -xe` to script for verbose output and error messages

**Review by shaikh-rashid — CHANGES_REQUESTED** [review]
@jagdeepsason2021 minor changes

File: htsget.sh [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+# Check if command exists
+if ! command -v python3 &>/dev/null; then
```
Comment (shaikh-rashid) [inline htsget.sh]: Not needed if you are already checking for python3.6

Comment (jagdeepsason2021) [reply htsget.sh]: Added set -xe and also removed the requested code from the script.

File: htsget.sh (hunk 1 above)
Comment (jagdeepsason2021) [reply htsget.sh]: Added the script as requested.

**Review by daisieh — APPROVED** [review]

**Pull request closed without merging**